tensorflow_addons/layers/gelu_test.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for GeLU activation."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras import backend as K
+from tensorflow_addons.layers.gelu import GeLU
+from tensorflow_addons.utils import test_utils
+from absl.testing import parameterized
+
+
+@parameterized.parameters([np.float16, np.float32, np.float64])
+@test_utils.run_all_in_graph_and_eager_modes
+class GELUTest(tf.test.TestCase):
+    def random_test(self, dtype):
```

Comment:
Seems that most of tests in addons use naming convention of `class TestGeLU`  and `def test_random`.  Also, there might not be any random part in the test. Should we rename it? Maybe `test_gelu` is enough.

Reply:
Yeah seems fair. I will make the necessary changes.